cmd/convox/install_test.go

```diff
@@ -42,6 +42,48 @@ func TestConvoxInstallSTDINCredentials(t *testing.T) {
 	)
 }
 
+func TestConvoxInstallValidateStackName(t *testing.T) {
+	stackId := "arn:aws:cloudformation:us-east-1:123456789:stack/MyStack/aaf549a0-a413-11df-adb3-5081b3858e83"
+	cycles := []awsutil.Cycle{
+		awsutil.Cycle{
+			awsutil.Request{"/", "", "/./"},
+			awsutil.Response{200, `<CreateStackResult><StackId>` + stackId + `</StackId></CreateStackResult>`},
+		},
+		awsutil.Cycle{
+			awsutil.Request{"/", "", ""},
+			awsutil.Response{200, ""},
+		},
+	}
+
+	handler := awsutil.NewHandler(cycles)
+	s := httptest.NewServer(handler)
+	defer s.Close()
+
+	os.Setenv("AWS_ENDPOINT", s.URL)
+
+	latest, _ := version.Latest()
+
+	test.Runs(t,
+		test.ExecRun{
+			Command: "convox install --stack-name valid",
+			Exit:    0,
+			Env:     map[string]string{"AWS_ENDPOINT_URL": s.URL, "AWS_REGION": "test"},
+			Stdin:   `{"Credentials":{"AccessKeyId":"FOO","SecretAccessKey":"BAR","Expiration":"2015-09-17T14:09:41Z"}}`,
+			Stdout:  Banner + "\nInstalling Convox (" + latest + ")...\n" + stackId + "\n",
+		},
```

Comment:
I believe you can put multiple runs in a test.Runs command. would save a couple lines of code